routing/router.go

```diff
@@ -2000,6 +2009,38 @@ func (r *ChannelRouter) processSendError(paySession *paymentSession,
 	}
 }
 
+// processSendToRouteError processes an error resulting from a SendToRoute
+// payment attempt. As there is no mission control and payment session involved
+// in SendToRoute (lnd is effectively remote controlled), error processing is
+// limited to applying channel updates if present. It is nothing more than
+// another source of gossip.
+func (r *ChannelRouter) processSendToRouteError(route *route.Route, err error) {
+	fErr, ok := err.(*htlcswitch.ForwardingError)
+	if !ok {
+		return
```

Comment:
Log or pass the error up perhaps?

Reply:
This function has been removed now, but the comment could apply to the regular one `processSendError` too. The error doesn't need to be passed up, because the caller already has the error. This function just interprets the error and makes calls to mission control if necessary.